viewer_clients/web-client/scripts/codecheckerviewer/FileViewBC.js

```diff
@@ -119,9 +119,11 @@ return declare(BorderContainer, {
           that.viewedFile   = item.filePath;
           that.viewedFileId = item.fileId;
 
-          CC_SERVICE.getSourceFileData(fileId, true, function(sourceFileData) {
+          CC_SERVICE.getSourceFileData(that.viewedFileId, true, function(sourceFileData) {
             editor._setContentAttr(sourceFileData.fileContent);
```

Comment:
Again: A thrift call may fail. On error it should log an error message to the JavaScript console.